Formula/l/lexido.rb

```diff
@@ -0,0 +1,21 @@
+class Lexido < Formula
+  desc "Innovative assistant for the command-line"
+  homepage "https://github.com/micr0-dev/lexido"
+  url "https://github.com/micr0-dev/lexido.git", tag: "v1.3.1", revision: "36f02d699f2709f0bd51a5f119e0670845b4ad9f"
+  license "AGPL-3.0-or-later"
+
+  depends_on "go" => :build
+
+  def install
+    # Set GO111MODULE to "auto" to enable Go modules
+    ENV["GO111MODULE"] = "auto"
+
+    # Build the binary using the default go build command
+    system "go", "build", *std_go_args, "-o", "#{bin}/lexido"
+  end
+
+  test do
+    # Perform a basic sanity check
+    assert_match "Lexido Command Line Tool v1.3.1", shell_output("#{bin}/lexido --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
This uses a api key to use it so the only test i could do is version

Reply:
> If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.

Reply:
im not sure if im missing something but i tried that and im now getting an error on here 

`Error: /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.1.0/bin/bundle clean
  ==> Testing lexido
  ==> /home/linuxbrew/.linuxbrew/Cellar/lexido/1.3.1/bin/lexido
  2024/04/07 20:51:06 exit status 1
  2024/04/07 20:51:06 Alas, there's been a Bubble Tea error: could not open a new TTY: open /dev/tty: no such device or address
  No API key found.
  Please visit https://aistudio.google.com/app/apikey to obtain your API key.
  Enter your API key here: API key set successfully for future sessions. `
  
 when you run lexido without a api get you get a asked to enter a key and if you enter a wrong key it will say the key is wrong
 
 I have set that up in the fomula but still get the error im new to Brew  (first time using it) so im not to sure what im doing wrong.